src/DynamoCore/Graph/Nodes/NodeCategories.cs

```diff
@@ -420,8 +421,8 @@ internal static string MakeRelativePath(string basePath, string subjectPath)
             var documentUri = new Uri(basePath, UriKind.Absolute);
             var assemblyUri = new Uri(subjectPath, UriKind.Absolute);
 
-            var relativeUri = documentUri.MakeRelativeUri(assemblyUri);
-            var relativePath = relativeUri.OriginalString.Replace('/', '\\');
+            var relativeUri = documentUri.MakeRelativeUri(assemblyUri).OriginalString;
+            var relativePath = WebUtility.UrlDecode(relativeUri.Replace("+", "%2B")).Replace('/', '\\');
```

Comment:
@gregmarr thanks for the explanation! Makes a lot more sense now.

* can you please add ``` MakeRelativePath should not return a URL encoded path.
new Uri() results in a URL encoded path.
Therefore, to undo that, we need to call UrlDecode on it.
Also, UrlDecode will convert + to space, but the Uri creation doesn't encode + as %2B.
In order to avoid + in the filename being converted to space, we need to encode + as %2B before calling it.```

 as a comment above the method call in **make relative path**

There are local UI tests in` DynamoCoreWPFTests` There are also utility tests for make relative path.